NEWS

```diff
@@ -10,6 +10,11 @@ CHANGES WITH 256 in spe:
           section, then all assigned VLAN IDs on the interface that are not
           configured in the .network file are removed.
 
+        * systemd-networkd's proxy support gained a new option to configure
```

Comment:
Could you insert section title? The first entry is about backward incompatibility. But this one is a new feature. So,
```
Network Management:

* systemd-networkd's proxy support ...
```

Reply:
Done.